### PR TITLE
BaSP-TG-31: Create seeds: Superadmin, Timesheet and Employee

### PR DESCRIPTION
**Pull request by matiasraviola**
All the seeds (Superadmin-Timesheet-Employee) have data to be able to test respecting the formats of each field.

### CODE REVIEW

File: src/seeds/timesheets.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  date: '1/1/2022',
+  task: 'morbi',
+  employee: mongoose.Types.ObjectId('6354438cfc13ae204b000061'),
+  project: mongoose.Types.ObjectId('6354438cfc13ae204b000062'),
+  hours: 81,
+}, {
+  _id: mongoose.Types.ObjectId('6354438cfc13ae204b000063'),
+  description: 'orci mauris',
+  date: '2/3/2022',
+  task: 'sed',
```
Comment (ignacioaristo) [inline src/seeds/timesheets.js:15]: Task must be FE or BE

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/seeds/timesheets.js (hunk 1 above)
Comment (matiasraviola) [reply src/seeds/timesheets.js]: done

File: src/seeds/timesheets.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import mongoose from 'mongoose';
+
+export default [{
+  _id: mongoose.Types.ObjectId('6354438cfc13ae204b000060'),
+  description: 'vitae nisl aenean',
+  date: '1/1/2022',
+  task: 'BE',
```
Comment (ignacioaristo) [inline src/seeds/timesheets.js:7]: Now it's okey, but we need the mongoose id for this.

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/seeds/timesheets.js (hunk 2 above)
Comment (matiasraviola) [reply src/seeds/timesheets.js]: Done

**Review by ignacioaristo — APPROVED** [review]

**Pull request merged**